web_shortcut/static/src/js/web_shortcut.js

```diff
@@ -6,6 +6,7 @@ odoo.define('web.shortcut', function (require) {
         WebClient = require('web.WebClient'),
         ViewManager = require('web.ViewManager'),
         ActionManager = require('web.ActionManager'),
+        Menu = require('web.Menu'),
```

Comment:
hello @tarteo 
we spent a lot of time on this one too and with your last adding feature (https://github.com/OCA/web/pull/565/commits/4faa3bf6e09a1272bba41b54aa3752634612ab78) it's like reverting our contribution as this commit implies an inconsistency with enterprise version.
As you have no more time I suggest you to add this functionality into another module (event a local module to you) that inherit of this one. Many Thanks in advance